packages/flutter/test/widgets/two_dimensional_scroll_view_test.dart

```diff
@@ -323,5 +323,104 @@ void main() {
       expect(scrollable.widget.diagonalDragBehavior, DiagonalDragBehavior.weightedContinuous);
       expect(scrollable.widget.dragStartBehavior, DragStartBehavior.down);
     }, variant: TargetPlatformVariant.all());
+
+    testWidgets('Fling and tap to stop', (WidgetTester tester) async {
```

Comment:
Can you update these tests to check two more things? These should verify they we are actually scrolling when these taps occur. Can you add the following:

- check the ScrollActivity.velocity in both axes when the test is about to tap and after.
  - the activity is available through the ScrollPosition
  - verifies we went from scrolling to stopping in response
- we should also verify the position is actually changing/stops changing as a result of flinging and then stopping it with a tap.
  - verify the ScrollPosition.pixels before flinging, before tapping, and after tapping